node/src/reactor/participating.rs

```diff
@@ -1020,11 +1039,13 @@ impl reactor::Reactor for Reactor {
                 Effects::new()
             }
             Event::DeployGossiperAnnouncement(GossiperAnnouncement::FinishedGossiping(
-                gossiped_deploy_id,
+                _gossiped_deploy_id,
             )) => {
-                let reactor_event =
-                    Event::BlockProposer(block_proposer::Event::BufferDeploy(gossiped_deploy_id));
-                self.dispatch_event(effect_builder, rng, reactor_event)
+                // let reactor_event =
```

Comment:
Any reason we need to keep this commented out code around?

Reply:
Not really I suppose, but we'll be re-instating the commented code very soon.  Either way, I'd like to keep this changeset identical to the one it's ported from unless you have strong objections (which it seems from subsequent comments you don't :) )